vagrant/README.md

```diff
@@ -124,3 +124,11 @@ Now start things up, but specify the aws provider:
 Your instances should get tagged with a name including your hostname to make
 them identifiable and make it easier to track instances in the AWS management
 console.
+
+### Libvirt with kvm ###
+
+Install the `vagrant-libvirt` plugin to provide libvirt and kvm support:
+```vagrant plugin install vagrant-libvirt``` or ```yum install vagrant-libvirt``` for fedora >=21
```

Comment:
it is depend of linux distributive.
if installed from official fedora repository - all dependencies are resolved.
if not - in vagrant-libvirt README a lot specific information.
Should i copy paste it to this readme or just add link ?